src/vm/mscorlib.h

```diff
@@ -889,6 +889,16 @@ DEFINE_METHOD(UNHANDLED_EVENTARGS,  CTOR,                   .ctor,
 DEFINE_CLASS(FIRSTCHANCE_EVENTARGS,   ExceptionServices,      FirstChanceExceptionEventArgs)
 DEFINE_METHOD(FIRSTCHANCE_EVENTARGS,  CTOR,                   .ctor,                      IM_Exception_RetVoid)
 
+DEFINE_CLASS_U(Loader,             AssemblyLoadContext,           AssemblyLoadContextBaseObject)
+DEFINE_FIELD_U(_unloadLock,                    AssemblyLoadContextBaseObject, _unloadLock)
+DEFINE_FIELD_U(ResolvingUnmanagedDll,          AssemblyLoadContextBaseObject, _resovlingUnmanagedDll)
+DEFINE_FIELD_U(Resolving,                      AssemblyLoadContextBaseObject, _resolving)
+DEFINE_FIELD_U(Unloading,                      AssemblyLoadContextBaseObject, _unloading)
+DEFINE_FIELD_U(<Name>k__BackingField,          AssemblyLoadContextBaseObject, _name)
```

Comment:
It would be better to have explicitly named field for this (it is what we do everywhere else).